memoizee/memoizee-tests.ts

```diff
@@ -0,0 +1,60 @@
+import memoize = require('memoizee');
+
+var fn = function (one:any, two:any, three:any) { /* ... */ };
```

Comment:
Would be better to test with more specific argument types, like `one: number, two: string`.